src/tracks/InstrumentTrack.cpp

```diff
@@ -1148,6 +1174,29 @@ class fxLineLcdSpinBox : public LcdSpinBox
 			gui->fxMixerView()->setFocus();// set focus to fxMixer window
 			//engine::getFxMixerView()->raise();
 		}
+
+		virtual void contextMenuEvent( QContextMenuEvent* event )
+		{
+			// for the case, the user clicked right while pressing left mouse-
+			// button, the context-menu appears while mouse-cursor is still hidden
+			// and it isn't shown again until user does something which causes
+			// an QApplication::restoreOverrideCursor()-call...
+			mouseReleaseEvent( NULL );
+
+			QPointer<CaptionMenu> contextMenu = new CaptionMenu( model()->displayName(), this );
+
+			// This condition is here just as a safety check, fxLineLcdSpinBox is aways
+			// created inside a TabWidget inside an InstrumentTrackWindow
+			if ( InstrumentTrackWindow* window = dynamic_cast<InstrumentTrackWindow*>( (QWidget *)this->parent()->parent() ) )
```

Comment:
@badosu So I looked up some other instances and we seem to instantiate and dispose these ContextMenus on-the-fly in most other places.  Some examples:

 * [`ControllerView.cpp#L178`](https://github.com/LMMS/lmms/blob/master/src/gui/widgets/ControllerView.cpp#L178)
 * [`EffectView.cpp#L249`](https://github.com/LMMS/lmms/blob/master/src/gui/widgets/EffectView.cpp#L249)
 * [`AutomatableSlider.cpp#L65`](https://github.com/LMMS/lmms/blob/master/src/gui/widgets/AutomatableSlider.cpp#L65)

So I don't see any major problems with your code, although I'd still like to hear what others have to say. :+1: 

P.S. Thanks for writing this!!! :)